airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -380,9 +380,15 @@ def _exec_pod_command(self, resp, command: str) -> Optional[str]:
             resp.write_stdin(command + '\n')
             while resp.is_open():
                 resp.update(timeout=1)
-                if resp.peek_stdout():
-                    return resp.read_stdout()
-                if resp.peek_stderr():
-                    self.log.info("stderr from command: %s", resp.read_stderr())
+                res = None
+                while resp.peek_stdout():
+                    res = res + resp.read_stdout()
+                if res:
+                    return res
```

Comment:
I'm just noticing this now (and maybe it's related to how this API works that I'm not familiar with) but won't this return stop us from logging any of the stderr below? Or is it the case that only one or the other is returned (which seems odd to me, but I'm not very familiar with kubernetes).

Reply:
Do we get stdout and stderr in real scenario? I have seen in spark we can get both together. Then I should move this return part to in end.